doc/source/index.rst

```diff
@@ -117,8 +117,8 @@ The task 'runastrodriz' can be used to reproduce the same Drizzle processing tha
 
    Running Astrodriz <runastrodriz.rst>
 
-Enhanced Pipeline Products API
-------------------------------
+HLA API
```

Comment:
The Mission Offices have determined that this code should NOT be referred to as "HLA" code in the documentation.  This change can not be included based on their decision.